### ISSUE TITLE
YOLOv11 No trained model was found for this model version - Error when using in Label Assist

### ISSUE THREAD

**Issue by benoitlemaire** [roboflow/notebooks#362]
### Search before asking

- [x] I have searched the Roboflow Notebooks [issues](https://github.com/roboflow-ai/notebooks/issues) and found no similar bug report.

### Notebook name

https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/train-yolo11-object-detection-on-custom-dataset.ipynb

### Bug

I am experiencing the same issue as [this user](https://discuss.roboflow.com/t/cannot-access-web-model-in-my-app/4161), but with YOLOv11.

I trained my model using the provided notebook, and everything works fine in the notebook: training completes successfully, predictions work as expected, and I can deploy the model to Roboflow. I can also see the model in the versioning interface and even upload images for testing.

However, when I try to use it in Label Assist, I encounter the following network error:

```
“You must train a model on this version with Roboflow Train before you can use inference.”
“No trained model was found for this model version.”
```

This issue prevents me from using the model in my workflow. It seems similar to the problem described by the other user, which was resolved by downgrading certain TensorFlow and JAX-related packages.

Steps to Reproduce:

- Train a YOLOv11 model using the official notebook.

- Deploy the trained model to Roboflow.

- Attempt to use the model in Label Assist.

- Observe the error messages above.

Possible Solution:
The previous user resolved this issue by downgrading the following packages:

!pip install tensorflow==2.14.0
!pip install tensorboard==2.14.0
!pip install keras==2.14.0
!pip install tensorflow-gcs-config==2.14.0
!pip install pyarrow==9.0.0
!pip install jax==0.4.20
!pip install jaxlib==0.4.20

Would these package versions be compatible with YOLOv11 as well? Are there any other workarounds or official fixes?

Any help would be appreciated!

### Environment

- Google Colab for training
- Roboflow for deployment
- Chrome on Macos 15.3

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SolomonLake** [comment]
Hi there! I'm happy to let you know we've added support for YOLOv11 uploaded models in Label Assist. This is retroactive, so all previously uploaded models will now work as expected!
